modules/040-node-manager/docs/FAQ.md

```diff
@@ -396,3 +397,756 @@ When changing the CRI in the cluster, additional steps are required for the mast
 
 ## How to add node configuration step?
 Additional node configuration steps are set by custom resource `NodeGroupConfiguration`.
+
+## How to use containerd with Nvidia GPU support?
+
+Since using the Nvidia GPU requires a custom containerd configuration, it is necessary to create a node group with the type cri `Unmanaged`.
+
+```yaml
+---
+apiVersion: deckhouse.io/v1
+kind: NodeGroup
+metadata:
+  name: gpu
+spec:
+  chaos:
+    mode: Disabled
+  cri:
+    type: NotManaged
+  disruptions:
+    approvalMode: Automatic
+  nodeType: CloudPermanent
```

Comment:
```suggestion
  nodeType: CloudStatic
```
CloudPermanent — nodes from ProviderClusterConfiguration will be created via dhctl;